modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -672,7 +707,8 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
           )
         }
 
-        line"$name: " + tpeAndDefault
+        deprecationAnnotation(hints).appendUnlessEmpty(Line.space) +
```

Comment:
class fields are fun because they're single line (so I didn't do a line break). For this reason, if the deprecation note is present we need to append a space.

Reply:
If someone knows a better combination of words than "unless empty", I'll take it. `appendIfPresent` didn't have a nice ring to it.

Reply:
Maybe this would be good enough? `.appendIf(_.nonEmpty)(Line.space)`

Reply:
`appendIf` looks neat

Reply:
f8331897f3495a92440f6d7edd3d53cea4483788